application/controllers/Announcement.php

```diff
@@ -190,4 +248,18 @@ function remove()
             show_error("You must be an admin to delete an announcement");
         }
     }
+
+    function searchannouncement()
```

Comment:
Why are there two of these functions I'm assuming the one above is the working one

Reply:
yes, I forgot to delete the old one, it was my first attempt a while ago